lib/reform/form/orm.rb

```diff
@@ -0,0 +1,29 @@
+module Reform::Form::ORM
+  module Base
```

Comment:
Looks great, @arturictus ! One thing: I don't use `Base` as a name anymore, that's as redundant as `Manager` or `Object`. Just call it `ORM`. `Base` is one of the many things we should _not_ copy from Rails.